Magento_Cms/layout/cms_index_index.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="main" htmlClass="container home">
+            <container name="home-wrapper-top"
```

Comment:
Let's refactor this code by moving as much as you can to `phtml` templates (including CMS blocks loading)